redundant-bmc/src/manager.cpp

```diff
@@ -11,8 +11,14 @@
 namespace rbmc
 {
 
+const std::string failoverPath =
+    std::string{RedundancyInterface::namespace_path::value} + '/' +
```

Comment:
nit: cannot we just use instance path to get the BMC0 path?

Reply:
That isn't upstream so I removed all uses of it:
```
paths:
    - namespace: /xyz/openbmc_project/state
      segments:
          - name: BMC
            description: >
                The object representing _this_ BMC is always at bmc0.
            value: bmc0

```

Reply:
oh yeah...I referred wrong branch.